app/renderer/src/components/Icon.js

```diff
@@ -24,9 +24,19 @@ const iconImgText = compose(
 );
 
 const Icon = (props) => {
+
+  const iconTextStyle = props.icon.bgColor
+    ? compose(
+      iconImgTextBase,
+      props.icon.bgColor && style({
```

Comment:
Yup. It is required for an incoming update to `dext-emoji-plugin` as it will be using unicode character instead `:emoji_name:` format.